### PR TITLE
migrator: log any exception during migration

### PR DESCRIPTION
**Pull request by jalavik**
During `migrate_chunk` any exception is hidden. This patch at least shows any problems to the log so it is visible.

@kaplun

### CODE REVIEW

File: inspire/modules/migrator/tasks.py [hunk 1]
```diff
@@ -124,10 +125,12 @@ def migrate_chunk(chunk, broken_output=None, dry_run=False):
                 before_record_index.send(recid, json=json)
                 json.update({'_index': index, '_type': 'record', '_id': recid})
                 records_to_index.append(json)
-            except Exception:
+            except Exception as err:
+                current_app.logger.exception(err)
```
Comment (kaplun) [inline inspire/modules/migrator/tasks.py]: Down below in `create_record` I was using: 
```python
logger.exception("Error in elaborating record ID {}".format(recid))
```

this should use the native Celery logging. What would be best (it passes automatically the exception)

Comment (jalavik) [inline inspire/modules/migrator/tasks.py]: Indeed that covers `create_record` but not if some exception happens on lines `124-127`. The problem we saw happened there.

Comment (jalavik) [inline inspire/modules/migrator/tasks.py]: Ah, I see now what you meant :dancer:  SO I guess we can just the logger then to not mix things.

**Comment by kaplun** [comment]
I am including this into my PR

**Pull request closed without merging**